From 44b016f3d0fb1eca04bedb9f856b330282151d23 Mon Sep 17 00:00:00 2001
From: Peter Chen <peter.chen@freescale.com>
Date: Tue, 21 Jul 2015 13:32:23 +0800
Subject: [PATCH] MLK-11269-1 ARM: imx_v7_defconfig: update USB certification
 configurations

- Add CONFIG_USB_OTG_WHITELIST and CONFIG_USB_HCD_TEST_MODE
- Change CONFIG_USB_EHSET_TEST_FIXTURE from 'm' to 'y'

Signed-off-by: Peter Chen <peter.chen@freescale.com>
(cherry picked from commit 40f0d728339fb460a108bc8b133207dd5a17dda8)
---
 arch/arm/configs/imx_v7_defconfig | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/arch/arm/configs/imx_v7_defconfig b/arch/arm/configs/imx_v7_defconfig
index dbe78dd..3639100 100644
--- a/arch/arm/configs/imx_v7_defconfig
+++ b/arch/arm/configs/imx_v7_defconfig
@@ -296,9 +296,11 @@ CONFIG_SND_SOC_IMX_MC13783=y
 CONFIG_SND_SOC_IMX_HDMI=y
 CONFIG_SND_SOC_IMX_SI476X=y
 CONFIG_USB=y
+CONFIG_USB_OTG_WHITELIST=y
 CONFIG_USB_OTG_FSM=y
 CONFIG_USB_EHCI_HCD=y
 CONFIG_USB_EHCI_MXC=y
+CONFIG_USB_HCD_TEST_MODE=y
 CONFIG_USB_ACM=m
 CONFIG_USB_STORAGE=y
 CONFIG_USB_CHIPIDEA=y
@@ -308,7 +310,7 @@ CONFIG_USB_SERIAL=m
 CONFIG_USB_SERIAL_GENERIC=y
 CONFIG_USB_SERIAL_FTDI_SIO=m
 CONFIG_USB_SERIAL_OPTION=m
-CONFIG_USB_EHSET_TEST_FIXTURE=m
+CONFIG_USB_EHSET_TEST_FIXTURE=y
 CONFIG_NOP_USB_XCEIV=y
 CONFIG_USB_MXS_PHY=y
 CONFIG_USB_GADGET=y
-- 
1.8.0

